membership-attribute-service/app/actions/CommonActions.scala

```diff
@@ -1,5 +1,5 @@
 package actions
-import akka.stream.Materializer
+import org.apache.pekko.stream.Materializer
```

Comment:
play 3 uses pekko which is essentially an open source fork of akka in a new package name.